tutorial03_traceable_layers/layerlib/CMakeLists.txt

```diff
@@ -15,18 +15,18 @@ include_directories(${SRC_DIR}/include
                     ${SRC_DIR}/layers
                     ${SRC_DIR}/loader)
 add_library(layer_utils STATIC
-        ${LAYER_DIR}/common/vk_layer_config.cpp
-        ${LAYER_DIR}/common/vk_layer_extension_utils.cpp
-        ${LAYER_DIR}/common/vk_layer_utils.cpp)
+        ${SRC_DIR}/layers/vk_layer_config.cpp
```

Comment:
Having this indirect dependency seems challenging. Should we be grabbing the binaries here rather than build from source?